docs/release-notes/mia-platform-v13.mdx

```diff
@@ -7,3 +7,140 @@ image: "img/release-note-link-preview.png"
+
+- Democratizing access to the IDP's internal tools and utilities
+- Providing user-friendly interfaces to facilitate configuration, management and monitoring of your composable software
+- Enriching the management of IDP with even more functionalities
+
+Let's go in deep with the new Features:
+
+### Enrich Meta Data and list all your data assets: Data Catalog
+
+Let's welcome our first release of **Data Catalog**, designed to provide the data governance team and data analysts with a centralized location where all metadata concerning the data assets of an organization are collected.
+
+ADD IMAGE -> dove facciamo vedere lo schemino architetturale
+
+Set up our Data Catalog Agent, to connect to different external sources, such as Oracle Database, Salesforce, Postgres Database, MySQL Database, Microsoft SQL Server Database, and to extract all metadata existing on your organization's System of Records. 
+
+Through Mia-Platform Console, it is possible to:
+
+- Keep metadata up to date with the external System of Records
+- Navigate data models in a centralized place
+- Enrich metadata with descriptions and GDPR regulation properties for each table field
+- Export your data models in .csv format
+- Make metadata usable via Data Catalog API from third-party systems and custom frontends
+
+To learn more, please refer to the [documentation](link).
+
+### Seamless Runtime Management with Fast Data Control Plane
+
+Mia-Platform Fast Data has been empowered with a Runtime Management solution, **Fast Data Control Plane**.  
+
+ADD IMAGE
+
+This Fast Data suite component provides a general overview of the Fast Data pipelines released on the runtime environment, and allows to monitor and govern the execution of your Fast Data system.  
+From its user-friendly web interface, you can pause and resume the consumption of data streams along Fast Data pipelines with just a few clicks to strategically allocate resources consumption and efficiently utilize them during peak demand processes.
+Many are the benefits of adopting the Fast Data control plan in your Fast Data Projects, for example:
+
+- **Optimize Resource Allocation**, by strategically pausing and resuming the consumption of data streams along the Fast Data pipelines
+- **Initial Load and Full Refresh processes management**, thanks to a user-friendly interface and without altering the configuration of your workloads deployed in runtime 
+- **Performance Testing and Simulation**, to simulate different scenarios for performance testing by observing your system behavior under various conditions
+- **Enhanced System Reliability**, allowing you to gracefully handle unexpected situations and/or system maintenance periods while minimizing system downtime.
+
+To learn more, please refer to the [documentation](link).
+
+### Streamlined how you build your Saga, now No-Code
+
+Let's welcome a new brand section: the Flow Manager Configurator!
+
+ADD IMAGE  
```

Comment:
If you want we can embedd from YT a short video of rhea explaining the new flow manager

Reply:
this would be great! Do you have the link?

Reply:
https://youtu.be/P6KesZaLNDM 

For now it is unlisted 'cuz we're missing description and subs. I ask to fix it as soon as possible.